crypto/cipher/null_cipher.c

```diff
@@ -94,63 +94,57 @@ static srtp_err_status_t srtp_null_cipher_dealloc (srtp_cipher_t *c)
     srtp_crypto_free(c);
 
     return srtp_err_status_ok;
-
 }
 
-static srtp_err_status_t srtp_null_cipher_init (void *cv, const uint8_t *key)
+static srtp_err_status_t srtp_null_cipher_init(void *cv, const uint8_t *key)
 {
-	/* srtp_null_cipher_ctx_t *c = (srtp_null_cipher_ctx_t *)cv; */
+    /* srtp_null_cipher_ctx_t *c = (srtp_null_cipher_ctx_t *)cv; */
 
     debug_print(srtp_mod_cipher, "initializing null cipher", NULL);
 
     return srtp_err_status_ok;
 }
 
-static srtp_err_status_t srtp_null_cipher_set_iv (void *cv, uint8_t *iv, srtp_cipher_direction_t dir)
+static srtp_err_status_t
+srtp_null_cipher_set_iv(void *cv, uint8_t *iv, srtp_cipher_direction_t dir)
 {
-	/* srtp_null_cipher_ctx_t *c = (srtp_null_cipher_ctx_t *)cv; */
+    /* srtp_null_cipher_ctx_t *c = (srtp_null_cipher_ctx_t *)cv; */
     return srtp_err_status_ok;
 }
 
-static srtp_err_status_t srtp_null_cipher_encrypt (void *cv,
-                                            unsigned char *buf, unsigned int *bytes_to_encr)
+static srtp_err_status_t srtp_null_cipher_encrypt(void *cv,
+                                                  unsigned char *buf,
+                                                  unsigned int *bytes_to_encr)
 {
-	/* srtp_null_cipher_ctx_t *c = (srtp_null_cipher_ctx_t *)cv; */
+    /* srtp_null_cipher_ctx_t *c = (srtp_null_cipher_ctx_t *)cv; */
     return srtp_err_status_ok;
 }
 
 static const char srtp_null_cipher_description[] = "null cipher";
 
 static const srtp_cipher_test_case_t srtp_null_cipher_test_0 = {
-    0,    /* octets in key            */
-    NULL, /* key                      */
-    0,    /* packet index             */
-    0,    /* octets in plaintext      */
-    NULL, /* plaintext                */
-    0,    /* octets in plaintext      */
-    NULL, /* ciphertext               */
-    0,
-    NULL,
-    0,
-    NULL             /* pointer to next testcase */
+    0,                  /* octets in key            */
+    NULL,               /* key                      */
+    0,                  /* packet index             */
+    0,                  /* octets in plaintext      */
+    NULL,               /* plaintext                */
+    0,                  /* octets in plaintext      */
+    NULL,               /* ciphertext               */
+    0,    NULL, 0, NULL /* pointer to next testcase */
 };
```

Comment:
I did actually go through and fix _a lot_ of these manually since the comments were pretty much all over the place. Did miss out on this one though because it was just a tiny bit less horrible as compared to others.

Any place in the code where there are trailing aligned comments that look OK -> was fixed by hand. :-)